Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+
+RUN apt-get update 
+RUN apt-get install -y vim less build-essential dh-make
+RUN apt-get install -y vim libpcap-dev dh-systemd
+ADD . /rtpsniff-1.0
+WORKDIR /rtpsniff-1.0
+RUN dpkg-buildpackage -uc -us -b
+CMD cp ../rtpsniff_1.0-1_amd64.deb .
```

Comment:
Ik heb een beetje twijfels bij het zo hardcoden van de versienummers. We krijgen straks voor elke build een 1.0 en een changelog met 1.0-1.

Misschien iets met envvars doen? Zie ook build-asterisk shellscript in 80e30a9171159716dbf78f3025eb0e259b0bdc02 van asterdebvg repo? Tevens is daarbij nog een kort stukje readme over hoe je moet builden.